cabal.project

```diff
@@ -32,19 +32,19 @@ package large-generics
 
 package large-records
   tests: True
---  ghc-options: -Werror
+  ghc-options: -Werror
 
 package large-records-benchmarks
   tests: True
   ghc-options: -Werror
 
 package beam-large-records
   tests: True
-  -- ghc-options: -Werror
+  ghc-options: -Werror
```

Comment:
TBH, these are terrible for development. That's why I commented them out.

Reply:
And yet the codebase is now not warning free.

Reply:
... because CI doesn't verify it!

If you really want it add

```
-- Options for local packages
local-ghc-options: -Werror
```

to `cabal.haskell-ci`.

Reply:
Ah, good. Thanks :) Will do.